internal/cmd/local/local_test.go

```diff
@@ -99,39 +101,34 @@ func TestCheckAirbyteDir(t *testing.T) {
 }
 
 func TestValues_FileDoesntExist(t *testing.T) {
-	cmd := InstallCmd{Values: "thisfiledoesnotexist"}
-	err := cmd.Run(context.Background(), k8s.TestProvider, telemetry.NoopClient{})
+
+	var root InstallCmd
+	k, _ := kong.New(
+		&root,
+		kong.Name("abctl"),
+		kong.Description("Airbyte's command line tool for managing a local Airbyte installation."),
+		kong.UsageOnError(),
+	)
+	_, err := k.Parse([]string{"--values", "/testdata/thisfiledoesnotexist"})
```

Comment:
The code is relying on Kong's `existingfile` validation now, instead of having a manual check (for better or worse)